README.rst

```diff
@@ -1,148 +1,331 @@
+================================
 FIWARE Policy Manager GE - Facts
-____________________
+================================
 
 | |Build Status| |Coverage Status| |Pypi Version| |Pypi License|
 
-Description
-===========
+.. contents:: :local:
 
-This module is part of FIWARE Policy Manager.
+Introduction
+============
+
+This is the code repository for **FIWARE Policy Manager GE - Facts**, a server to process the incoming facts from the
+`Orion Context Broker <https://github.com/telefonicaid/fiware-orion>`__
+and publish the result into a RabbitMQ queue to be analysed by Fiware-Cloto. The facts are the result of the server
+resources consumption.
+
+This project is part of FIWARE_.
+Check also the `FIWARE Catalogue entry for Policy Manager`__
+
+__ `FIWARE Policy Manager - Catalogue`_
+
+
+Any feedback on this documentation is highly welcome, including bugs, typos or
+things you think should be included but aren't. You can use `github issues`__
+to provide feedback.
+
+__ `Fiware-facts - GitHub issues`_
+
+`Top`__.
+
+__ `FIWARE Policy Manager GE - Facts`_
+
+GEi overall description
+=======================
+Policy Manager is called Bosun.
```

Comment:
I think this has to be rephrased: something like "Bosun is the reference implementation of Policy Manager GE". Additionally, the "GEri" acronym could be used.